corehq/apps/app_manager/templates/app_manager/profile.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <profile version="{{ app.version }}"
-         update="{{ profile_url }}"
+         update="{{ profile_url }}&amp;profile={{ build_profile_id }}"
```

Comment:
Just confirming: `profile_url` always has a query string (so it already contains a `?`), and the ones below in this file never contain a query string. Is that correct?